extension/js/common/core/common.ts

```diff
@@ -92,7 +98,15 @@ export class Str {
   }
 
   public static asEscapedHtml = (text: string) => {
```

Comment:
This PR uncovered some duplicate effort - `Str.asEscapedHtml` and `Xss.escapeTextAsRenderableHtml` do more or less the same thing. Please combine them to a single method `Str.escapeTextAsRenderableHtml` (but in the Str class). 

Notice that they both replace slightly different set of characters, please combine so that it replaces any of the chars mentioned in either of the two current methods.

Careful about order of replacing, it is ordered deliberately so that characters like `&` don't get double-escaped.

Thanks!

Reply:
Done :+1: